xarray/core/parallelcompat.py

```diff
@@ -132,8 +132,9 @@ def get_chunked_array_type(*args) -> ChunkManagerEntrypoint:
         if chunkmanager.is_chunked_array(chunked_arr)
     ]
     if not selected:
+        possible_missing_package_name = type(chunked_arr).__module__.split(".")[0]
         raise TypeError(
```

Comment:
considering that the error is actually caused by a missing package, could we consider 
- using a more general exception, such as `RuntimeError`, instead of the current `TypeError`?
- or implementing a custom exception specifically for this purpose?